browser-api/src/main/java/com/duckduckgo/app/usage/app/AppDaysUsedRepository.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2024 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.usage.app
+
+import java.util.*
+
+/**
+ * Repository for storing and retrieving the number of days the app has been used
+ */
+interface AppDaysUsedRepository {
```

Comment:
QQ: Are we planning to revert this once survey completed?

sgtm now to unblock and release the autofill survey. However, if final, we should discuss the api change to validate what makes sense to expose, how, where, etc.

Reply:
Wasn't planning to revert, as the changes seem generally useful.

>we should discuss the api change

that was why it was added to the API proposal a few weeks back. 

>sgtm now to unblock and release the autofill survey

Ok, I will carry on with this as planned and can discuss if there should be additional changes to the API made afterwards.